packages/radix-vue/src/Tabs/TabsTrigger.vue

```diff
@@ -19,30 +19,32 @@ const props = withDefaults(defineProps<TabsTriggerProps>(), {
 })
 const context = inject(TABS_INJECTION_KEY)
 
-const triggerId = computed(() => makeTriggerId(context!.baseId, props.value))
-const contentId = computed(() => makeContentId(context!.baseId, props.value))
+const ids = computed(() => ({
+  triggerId: makeTriggerId(context!.baseId, props.value),
+  contentId: makeContentId(context!.baseId, props.value),
+}))
 
```

Comment:
Can we maintain 2 computed? Cause in some case we are providing this value, and I would like to keep this consistent as others 😁

Reply:
I don't know why, I have already submitted the recovered code in my fork repository, but that commit has not been added to this PR. Did I do it incorrectly？